catboost/python-package/ut/medium/test.py

```diff
@@ -3664,3 +3664,37 @@ def test_eval_set_with_no_target_with_eval_metric(task_type):
     )
     with pytest.raises(CatboostError):
         model.fit(train_pool, eval_set=eval_set_pool)
+
+
+def test_model_comparison():
+    def fit_model(value):
+        pool = Pool(TRAIN_FILE, column_description=CD_FILE)
+        model = CatBoostClassifier(iterations=5)
+        model.fit(pool)
+        return model
+
+    model0 = CatBoostClassifier(iterations=5)
+    model1 = fit_model(42)
+    model2 = fit_model(-999)
+
+    # Test checks that model is fitted.
+    try:
```

Comment:
canonical way for pytest is 

```py
with pytest.raises(CatboostError):
    ...
```

like that: https://github.com/catboost/catboost/blob/1aa0ba06cceef82cd4dc64cad084b293ae143839/catboost/python-package/ut/medium/test.py#L539